lms/resources/_js_config.py

```diff
@@ -10,7 +10,7 @@
 from lms.views.helpers import via_url
 
 
-class JSConfig:  # pylint:disable=too-many-instance-attributes
```

Comment:
Lots of no-longer-needed comments removed thanks to the newly-enabled `useless-suppression` warning